tests/specs/graph/JsrSpecifiers_FastCheck_TsModule.txt

```diff
@@ -71,3 +71,11 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
   }
   class Public1 {
   }
+  --- DTS ---
+  export module Test.Test {
+    export class MyClass extends Public1 {
+      prop!: string;
```

Comment:
We'll need to remove the `!` here.

Reply:
Was an symptom of not traversing into TypeScript namespace.